src/malloc/mallocng/glue.h

```diff
@@ -20,9 +20,9 @@
 #define is_allzero __malloc_allzerop
 #define dump_heap __dump_heap
 
-#define malloc __libc_malloc_impl
-#define realloc __libc_realloc
-#define free __libc_free
+//#define malloc __libc_malloc_impl
+//#define realloc __libc_realloc
+//#define free __libc_free
```

Comment:
Is this required? If so just remove the lines, there is no reason to keep commented dead code around

Reply:
I left it like this throughout musl, because I felt that there should be a better way, and to avoid those defines coming back sneakily during a rebase. the better thing would actually be a #ifndef __CHEERP__ around them

Reply:
I changed this to Yuri's suggestion.